src/fsharp/PackageManager.Integration.fs

```diff
@@ -33,12 +33,20 @@ let GetPaketLoadScriptLocation baseDir optionalFrameworkDir =
 let GetCommandForTargetFramework targetFramework =
     ReferenceLoading.PaketHandler.MakePackageManagerCommand "fsx" targetFramework
 
-let RegisteredPackageManagers = ["paket:"] // this should really be records with names and load script names and stuff
+type PackageManager = {
```

Comment:
@forki are you trying to trigger our collective YAGNI sensor? if we have paket, what else do we need? :laughing: